docs/Analysis.rst

```diff
@@ -39,5 +39,7 @@ A particular mode can be forced by calling ``dh5view`` with the ``-m`` option, e
 
     dh5view -m LM filename
 
-
+Rendering
```

Comment:
```suggestion
Exporting colour-mapped images
```

I think the term "rendering" is a bit vague - especially for a biological reader. What we are doing here is exporting images with colour -mapping and scale bars etc ... This applies to all uses of rendering in the docs.